### ISSUE TITLE
So I tried the new build LIVE last night ... here's a field report.

### ISSUE THREAD

**Issue by deadpahn** [asiekierka/nitrotracker#27]
Overall, it was great. With OG NT loading songs would give the DOH! error. Did not get that last night thankfully. I did run into some weird behavior though that might be hard to reproduce.

First of all, I dunno what happened, but it just STOPPED at some point. Both screens went black, I picked it up, it went back to normal, and then went black again. No sure if it's mechanical failure of the DS or what. It's in pretty good shape, almost brand new condition so I dunno. I'll try to reproduce it but I'm not sure what happened. The video shows that moment.

HOWEVER, it's a lot easier to recover from that than when OG NT used to crash, holy jeez don't even get me started. 

The second issue was when I was working on files on the train to Toronto from Montreal. Songs would get stuck in the SAVING prompt. After rebooting the songs were messed up and wouldn't load (looked like a blank file when I popped the card in a computer). Old NT used to do this too so I think this isn't regression (I actually was thinking about if I could set up some sort of selenium type of thing for regression testing lol). The only thing I can think of is I was loading some LONGER wav files (drum loops and a few cheesy voice clips)? Anyway, after several broken attempts I ended up cleaning things up with milkytracker instead which worked great.

Even though this sounds sketchy, I can assure you that OG was way worse live before. The improvements added more stability than subtracted. Having to reboot into an r4, see that boot screen etc was always a vibe lol. 

Here's a small video, I'll post the whole thing later but everyone had a blast and the new NT build worked great.

https://user-images.githubusercontent.com/1032331/227749857-5a674ccb-ab8a-4605-b70e-74bad9d7025a.mp4

**Comment by asiekierka** [comment]
>First of all, I dunno what happened, but it just STOPPED at some point. Both screens went black, I picked it up, it went back to normal, and then went black again.

This is going to sound completely out of left field, but was there anything magnetic nearby? It could trigger the DS Lite's lid sensor, making it think that it should go to sleep.

**Comment by deadpahn** [comment]
Yes. There was a stack of fridge magnets on the opposite end of the table.
Maybe one of them got too close? I'll keep an eye on this for next time.

**Comment by asiekierka** [comment]
The second issue might be improved with 0.4.3/0.4.4, but, no guarantees.

**Comment by asiekierka** [comment]
@deadpahn I have no other place to reach you, so apologies - but what do you think, from an ergonomics perspective, of using the "Y" or "R" buttons as a modifier (think like a Shift key)? What do you think about it working as a toggle, versus working as a held trigger? I want to add more features to the sample editor, but there's not much room to add more buttons, so I was thinking of adding a physical modifier button to be able to place two buttons in the place of one, for example:

* the "fade in sample" button becomes a "fade out sample" button if the modifier is active,
* the "trim sample to selection" button becomes a "delete selection from sample" button if the modifier is active,
* etc.

**Comment by deadpahn** [comment]
"I have no other place to reach you, so apologies"
no problem!

"what do you think, from an ergonomics perspective, of using the "Y" or "R" buttons as a modifier"
So ... here's the thing. tl;dr I've been scooping up as many DSs as I can to make sure I never run into any issues in the future for my act. I'm at around 16 or 17 now. The one thing I noticed is this: the triggers usually are the ones to break first. The triggers on the DS are rough. Sometimes it's an old Nintendo "blow in the cart" sort of fix, but other times the damage is more permanent.  Thankfully NT has the toggle button you can tap to switch screens. So, I would opt for using the front facing buttons, only as a shortcut, having on screen controls always being the fall back in case the DS is borked.

I think this is a design philosophy that could future proof use for years to come. UNLIKE THE ORIGINAL GB and it's LSDJ/NANO LOOP scene ... I'm not sure how "fixable" the DS is. Where the original DMG was trivial to DIY/MOD (which extended it's life for years after it's intended use) the inside of the DS is a bit hairy. I've botched a few fixes myself lol. When possible, I would opt for on onscreen equivalent. 
 
"(think like a Shift key)? What do you think about it working as a toggle, versus working as a held trigger?"
Both seem good for different uses/features cases.

"I was thinking of adding a physical modifier button to be able to place two buttons in the place of one"
A "next page" on screen arrow/button that can also be toggled with a button seems like it would work out. Going back to accesibility, "holding" a button in a modifier might introduce some ergo problems. Maybe I'm just hyper paranoid of the contacts on the DS in the first place lol but I think "next page" arrow and a toggle back-forth button would be leet.

**Comment by asiekierka** [comment]
>UNLIKE THE ORIGINAL GB and it's LSDJ/NANO LOOP scene ... 

Hah! Unrelated, but I want to create a tracker for the WonderSwan one day...

**Comment by deadpahn** [comment]
Gunpei Yokoi would be proud lol
